python/lsst/daf/butler/core/datasets.py

```diff
@@ -46,7 +45,7 @@ class DatasetType(object):
     All arguments correspond directly to instance attributes.
```

Comment:
Could really do with explicit Attributes and Parameters section.

Reply:
I thought it was policy to document these in the property docstrings?

Reply:
Yes. I thought that in this case there was a difference between init parameters and attributes (on phone so not checking).